compiler/src/iree/compiler/Codegen/Common/MaterializeEncodingIntoPadding.cpp

```diff
@@ -293,9 +204,7 @@ struct MaterializeEncodingIntoPaddingPass final
     // with the exception of a few ops that have to account for padding.
     // We add custom patterns with much higher priority to run before the
     // equivalent 'Nop' patterns.
-    materializeEncodingPattern.add<MaterializeFlowDispatchTensorLoadOp,
-                                   MaterializeFlowDispatchTensorStoreOp,
-                                   MaterializeInterfaceBindingEncoding>(
+    materializeEncodingPattern.add<MaterializeInterfaceBindingEncoding>(
```

Comment:
The removal of `MaterializeInterfaceBindingEncoding` requires some more work; I'll work on it in a follow-up.